Snowflake.Data/Core/HttpUtil.cs

```diff
@@ -173,6 +177,11 @@ internal RetryHandler(HttpMessageHandler innerHandler) : base(innerHandler)
                 HttpResponseMessage response = null;
                 int backOffInSec = 1;
 
+                ServicePoint p = ServicePointManager.FindServicePoint(requestMessage.RequestUri);
+                p.Expect100Continue = false; // Saves about 100 ms per request
+                p.UseNagleAlgorithm = false; // Saves about 200 ms per request
+                p.ConnectionLimit = 20;      // Default value is 2, we need more connections for performing multiple parallel queries

```

Comment:
Sigh. this looks ugly, but I also don't find a good way to enforce that. My only questions is this code will be executed for every request right? Not just the retried requests right?

Reply:
Yes, it will since it's in the SendAsync function. But I suspect that something similar is done under the hood using the global parameter anyway.